src/satosa/frontends/saml2.py

```diff
@@ -343,8 +348,9 @@ def _handle_authn_response(self, context, internal_response, idp):
             name_qualifier=None,
         )
 
-        dbgmsg = "returning attributes %s" % json.dumps(ava)
-        satosa_logging(logger, logging.DEBUG, dbgmsg, context.state)
+        msg = "returning attributes {}".format(json.dumps(ava))
+        logline = lu.LOG_FMT.format(id=get_session_id(context.state), message=msg)
```

Comment:
you've probably seen this from the failed tests but for the sake of obviousness this needs to be written as `lu.get_session_id`

Reply:
Thanks.